app/multitenant/dynamo_collector.go

```diff
@@ -78,6 +81,24 @@ var (
 		Name:      "nats_requests",
 		Help:      "Number of NATS requests.",
 	}, []string{"method", "status_code"})
+
+	memcacheHits = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "memcache_hits",
+		Help:      "Reports that missed our in-memory cache but went to our memcache",
+	})
+
+	memcacheMiss = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: "scope",
+		Name:      "memcache_miss",
+		Help:      "Reports that missed both our in-memory cache and our memcache",
+	})
+
+	memcacheRequestDuration = prometheus.NewSummaryVec(prometheus.SummaryOpts{
+		Namespace: "scope",
+		Name:      "memcache_request_duration_nanoseconds",
+		Help:      "Time spent doing memcache requests.",
+	}, []string{"method", "status_code"})
```

Comment:
Given that we use `GetMulti` for our memcache queries, would it work to label each request with num hits & num total? (i.e. we can't simply have a boolean hit-or-miss label)